.github/workflows/automatic-updates.yml

```diff
@@ -0,0 +1,53 @@
+name: Automatically update Docker image versions
+
+on:
+  schedule:
+    - cron: "*/15 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nodejs'
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          # to access `--experimental-fetch`
+          node-version: '17'
+
+      - name: Run automation script
+        run: node --experimental-fetch build-automation.mjs
+
+      - name: Get updated versions from updated_versions file and store in env vars
+        run: |
+          [ -f updated_versions ] && echo "UPDATED_VERSIONS=$(cat updated_versions)" >> GITHUB_ENV
+          rm -f updated_versions
```

Comment:
instead of this, can we use `output`? https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions. Something like


```js
console.log(`::set-output name= updated_versions::${versions.join(',')}`);
```

Reply:
OK.